docs/pages/blog/material-ui-v7-is-out.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Material UI v7 is out 🚀
+description: Material UI v7 is now stable, improving integration with modern tools and consistency across the library.
+date: 2025-03-31T12:00:00.000Z
+authors: ['diegoandai']
+tags: ['Material UI', 'Product']
+manualCard: true
+---
+
+Material UI v7 is now stable.
+This major improves the integration with modern tools and consistency across the library.
+It's a lean upgrade designed to be straightforward to upgrade to.
+
+## Improved ESM support
+
+The package layout has been updated to comply with ESM.
+This is achieved by adding the `exports` field to the `package.json` file.
+The `export` field indicates where bundlers can find CJS, ESM, and modern bundle files.
+You can read more about this in the [Node.js documentation](https://nodejs.org/api/packages.html#packages_exports).
+
+This update fixes several issues with popular bundlers like Vite and webpack.
+It also reduces the amount of setup required to work with these tools.
+
+## Opt-in support for CSS layers
```

Comment:
We backported this to v6, but I think it's still worth mentioning here as it relates to Tailwind v4. It was worked on during the v7 cycle anyway.